### PR TITLE
Remove unused enableSites() method

### PR DESCRIPTION
**Pull request by roland-d**
Pull Request for Issue #26189 

### Summary of Changes
Removes the unused method `enableSites()`

### Testing Instructions
1. Apply PR
2. Go to System -> Update Sites
3. Check that Enable and Disable of Update sites still works

### Expected result
Sites are disabled and enabled as expected.

### Documentation Changes Required
None

@brianteeman

### CODE REVIEW

File: administrator/components/com_installer/Model/UpdateModel.php [hunk 1]
```diff
@@ -306,41 +306,6 @@ public function purge()
...
-			->where($db->quoteName('enabled') . ' = 0');
-		$db->setQuery($query);
-
-		try
-		{
-			$db->execute();
-		}
-		catch (ExecutionFailureException $e)
-		{
-			$this->_message .= Text::_('COM_INSTALLER_FAILED_TO_ENABLE_UPDATES');
```
Comment (wilsonge) [inline administrator/components/com_installer/Model/UpdateModel.php:331]: Looks like this string can be removed too?

**Review by zero-24 — APPROVED** [review]

**Comment by uGE70** [comment]
I have tested this item :white_check_mark: successfully on 60334e28d29a7e07f18fbb106baadd3184fbe24f<br /><br />PR applied
Checked all update Sites
Enabled successfully by clicking on buttons or icons
Disabled successfully by clicking on buttons or icons
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26193">issues.joomla.org/tracker/joomla-cms/26193</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 60334e28d29a7e07f18fbb106baadd3184fbe24f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26193">issues.joomla.org/tracker/joomla-cms/26193</a>.</sub>

**Comment by zero-24** [comment]
Merging 👍

**Pull request merged**